### DISCUSSION TITLE
Lambda API REST - STATUS : 415

### DISCUSSION THREAD

**Discussion by biplab123321** [aws/aws-connected-device-framework#92] [Q&A]
The deployment CDF was successful. I was testing the rest call from API Gateway to do Greengrass provisioning.
The API generated for the Greengrass provisioning: has the error: Method completed with status: 415

attaching error let me know if I am missing something in the POST Call.

Request to let me know if missing something.

**Comment by boardthatpowder** [comment]
A `415` indicates unsupported media type. Looks like you are missing the `Accept` and `Content-Type` request headers. Specify these as `application/vnd.aws-cdf-v1.0+json` (see [walkthrough](https://github.com/aws/aws-connected-device-framework/tree/main/source/packages/services/greengrass2-provisioning)).

**Reply by biplab123321** [reply]
I added the header as mentioned still faces
![erroe2](https://user-images.githubusercontent.com/114371445/201749714-a31adb9f-65d0-41d6-99e5-c35c11be65a7.png)
 the same error.

**Reply by boardthatpowder** [reply]
Can you try removing the spaces after the `:` in your request headers. Looking at your logs in the right hand side the space appearing after `=` in the headers being passed to the function look suspect.

**Reply by biplab123321** [reply]
Currently getting 202 Status code:
please let me know if I am missing something
![202](https://user-images.githubusercontent.com/114371445/201960678-d036a927-ae25-47d4-b90e-c69e8934f6ca.png)
template file used: 
As shown in the link:https://github.com/aws/aws-connected-device-framework/tree/main/source/packages/services/greengrass2-provisioning
